lib/utils/cache/index.ts

```diff
@@ -36,12 +36,7 @@ if (config.cache.type === 'redis') {
         }
     };
     globalCache.set = (key, value, maxAge = config.cache.routeExpire) => {
-        if (!value || value === 'undefined') {
-            value = '';
-        }
-        if (typeof value === 'object') {
-            value = JSON.stringify(value);
-        }
```

Comment:
They have been there 7 years ago (a1f060f3d2205fbe5cfc1672ad2bb0212e350c65, 2257875db752c133c4b38424599e115cf959c4ff), and changing them obviously breaks the tests.